src/panels/lovelace/cards/hui-picture-entity-card.ts

```diff
@@ -136,14 +139,21 @@ class HuiPictureEntityCard extends LitElement implements LovelaceCard {
     }
 
     const domain = computeDomain(this._config.entity);
+    let image: string | undefined = this._config.image;
+    switch (domain) {
+      case "image":
+        image = computeImageUrl(stateObj as ImageEntity);
+        break;
+      case "person":
+        image = (stateObj as PersonEntity).attributes.entity_picture;
```

Comment:
Apparently, picture entity cards are already used for person entities. The default generated lovelace config adds picture entity cards for every known person. If that person entity doesn't have a picture, entity_picture is undefined and it should fallback to the this._config.image. 

```suggestion
        if ((stateObj as PersonEntity).attributes.entity_picture) {
          image = (stateObj as PersonEntity).attributes.entity_picture;
        }
```

Reply:
Fixed in 349b116e2a7ed9cebfa9f17da01e159b4efb9f16